README.md

```diff
@@ -1,6 +1,89 @@
 # safe-network-launch-tool
 A cross platform tool to easily launch a SAFE Network test section from home
 
+## Installing the SAFE Vault
+
+A local SAFE network is bootstrapped by running several [SAFE vaults](https://github.com/maidsafe/safe_vault) which automatically interconnect forming a network.
+
+In order to run your own local network you'd need to follow these steps:
+- Download latest release from [safe_vault releases](https://github.com/maidsafe/safe_vault/releases/tag/0.20.1)
```

Comment:
Probably best that the link here is changed to `https://github.com/maidsafe/safe_vault/releases/latest/`

Reply:
yes, true, I'll add that change